go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/disgoorg/disgo
 go 1.18
 
 require (
+	github.com/Microsoft/go-winio v0.5.2
```

Comment:
Isn't there a better package for Linux users as well?

Reply:
linux doesn't require an extra package. see https://github.com/disgoorg/disgo/blob/feature/rpc/rpc/pipe_unix.go